metricbeat/module/aerospike/namespace/namespace.go

```diff
@@ -82,14 +84,14 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	}
 
 	for _, node := range m.client.GetNodes() {
-		info, err := as.RequestNodeInfo(node, "namespaces")
```

Comment:
HAs "RequestNodeInfo" been deprecated in the latest version of aerospike ?

Reply:
Hi, yes, it has been removed from the API starting from version 2.0 of the library (release in March 2019). From the [release notes](https://github.com/aerospike/aerospike-client-go/releases?page=9):

> Moves RequestNodeInfo and RequestNodeStats to methods on Node object, and adds InfoPolicy to the relevant API signatures.